CONTRIBUTING.md

```diff
@@ -58,24 +60,19 @@ Make sure all the following are true when creating a pull-request:
 
 6. To test your changes on a cluster use
 
-    1. kubectl to connect to a cluster
+    > Pushing to the default container registry (`quay.io/dynatrace/dynatrace-operator`) requires specific permissions.
+    > You can use your own container registry by setting the `IMAGE` environment variable to a different value.
+
+    1. Connect to a cluster using `kubectl`
     2. Use make commands to build and deploy your operator as follows:
 
     ```sh
     make build && make deploy
     ```
 
-    >**NOTE:**
-    > When building on ARM machines (such as Apple M1) podman/docker uses the local architecture if not specified otherwise.
-    > To override set the ENV var OPERATOR_DEV_BUILD_PLATFORM to the desired platform (e.g. linux/amd64).
-    >
-    > ```shell
-    >    export OPERATOR_DEV_BUILD_PLATFORM=linux/amd64
-    > ```
-
 7. Create a pull request from the fork ([see guide](https://help.github.com/articles/creating-a-pull-request-from-a-fork/)), with a proper title and fill out the description template. Once everything is ready, set the PR ready for review.
 
-8. A maintainer will review the pull request and make comments. Prefer adding additional commits over amending and force-pushing since it can be difficult to follow code reviews when the commit history changes. Commits will be squashed when they're merged.
+8. A maintainer will review the pull request and make comments.Prefer adding additional commits over amending and force-pushing since it can be difficult to follow code reviews when the commit history changes. Commits will be squashed when they're merged.
```

Comment:
```suggestion
8. A maintainer will review the pull request and make comments. It's preferable to add additional commits over amending and force-pushing since it can be difficult to follow code reviews when the commit history changes. Commits will be squashed when they're merged.
```